htlcswitch/switch.go

```diff
@@ -245,8 +246,14 @@ type Switch struct {
 	// This will be retrieved by the registered links atomically.
 	bestHeight uint32
 
-	wg   sync.WaitGroup
-	quit chan struct{}
+	// TODO(yy): remove handleLocalResponseWG, once handleLocalResponse runs
```

Comment:
hmm why it's my TODO😂

Reply:
`handleLocalResponseWG` was added because `handleLocalResponse` is called in a goroutine, which can't be tracked using GoroutineManagaer. There is an existing [TODO(yy)](https://github.com/lightningnetwork/lnd/blob/c0e85d36ae0263f8b7283f8d0dd5e9f127f760f3/htlcswitch/switch.go#L3022-L3024) to remove the goroutine running `handleLocalResponse`. I copy-pasted that TODO here, since if that TODO is fixed, then handleLocalResponseWG is not needed, so this TODO is also fixed :-)